src/main/java/it/pagopa/selfcare/pagopa/backoffice/security/JwtAuthenticationFilter.java

```diff
@@ -18,19 +15,16 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 
-import static it.pagopa.selfcare.pagopa.backoffice.handler.RestExceptionsHandler.UNHANDLED_EXCEPTION;
 import static it.pagopa.selfcare.pagopa.backoffice.util.JwtUtil.getJwtFromRequest;
 
 @Slf4j
-@Component
```

Comment:
All components are registered automatically by Spring. We don't want to active this Filter during junit test.